backend/src/data_entry/mod.rs

```diff
@@ -419,81 +453,72 @@ mod tests {
             .await
             .expect("No data found");
         let data: PollingStationResults = serde_json::from_slice(&data.data.unwrap()).unwrap();
-        assert_eq!(data.voters_counts.poll_card_count, poll_card_count);
+        assert_eq!(
+            data.voters_counts.poll_card_count,
+            request_body.data.voters_counts.poll_card_count
+        );
 
         // Finalise data entry after correcting the error
         let response = save(pool.clone(), request_body.clone(), 1).await;
         assert_eq!(response.status(), StatusCode::OK);
-        let response = finalise_entry(pool.clone(), 1).await;
+        let response = finalise(pool.clone(), 1).await;
+        assert_eq!(response.status(), StatusCode::OK);
+
+        // Save a second data entry
+        let response = save(pool.clone(), request_body.clone(), 2).await;
         assert_eq!(response.status(), StatusCode::OK);
 
-        // Check if the first data entry was finalised:
-        // and that a second entry was created
-        /* TODO: Enable these asserts once the frontend supports second entry
+        // Check if the first data entry was finalised and that a second entry was created
         let rows = query!("SELECT * FROM polling_station_data_entries")
             .fetch_all(&pool)
             .await
             .unwrap();
+        println!("{:?}", rows);
```

Comment:
Do we want to keep this debug statement?

Reply:
Removed in 9305cac87a1473f3de70d1c9440d9eae89fadd39